packages/core-typings/src/ee/IAuditLog.ts

```diff
@@ -12,12 +12,13 @@ export interface IAuditLog extends IRocketChatRecord {
 	fields: {
 		type: string;
```

Comment:
Since we're already changing this type, can we be more specific on this one? It'd be good to have the new type you just created mapped somewhere

Reply:
Not sure about that. My changes makes Type a constant one but only for my case. When auditing messages Type can apparently be,any string (or room type i guess)

We should better look into this type or we'll create an unmantainable one if we add more audit types  👀